faststream/nats/subscriber/usecase.py

```diff
@@ -464,3 +486,123 @@ async def _consume_pull(self) -> None:
 
                 if messages:
                     await self.consume(messages)
+
+
+class KvWatchHandler(LogicSubscriber["KeyValue.Entry"]):
+
+    def __init__(
+        self,
+        *,
+        kv_watch: "KvWatch",
+        broker_dependencies: Iterable[Depends],
+        broker_middlewares: Iterable["BrokerMiddleware[List[Msg]]"],
+        # AsyncAPI args
+        title_: Optional[str],
+        description_: Optional[str],
+        include_in_schema: bool,
+    ) -> None:
+        parser = KvParser()
+
+        super().__init__(
+            subject="",
+            extra_options=None,
+            queue=None,
+            stream=None,
+            pull_sub=None,
+            no_ack=None,
+            retry=None,
+            kv_watch=kv_watch,
```

Comment:
You should not store this argument in the base classes if it is required in the current

Reply:
Mv it from `super().__init__(kw_watch=...)` to just `self.kw_watch=...`